axon-server/administration/cluster-template.md

```diff
@@ -0,0 +1,84 @@
+#Cluster template
```

Comment:
Don't forget to add this page to the `SUMMARY.md`, otherwise it will not show up in the index for our readers.

Reply:
Additionally, it should be added to an Axon Server specific section. Thinking `installation` would be the most likely, thus warranting another change in the `installation/README.md` file.

Reply:
I added it to configuration folder, next to the CLI, REST configuration